index.html

```diff
@@ -34,4 +34,194 @@ <h1>Kiprono Japhet</h1>
             <img src="images/image_geometry_2.png" alt="image">
         </div>
     </header>
+    <section class="projects">
```

Comment:
[REQUIRED] Nice job on using the semantic tag `<section/>` for the ` projects section` :+1: :sunglasses:. One little thing is missing though, I guess you forgot to give your section a `heading` as it is shown in the [chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Please refer to the comparison below:

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| ![Screenshot from 2022-02-17 23-19-36](https://user-images.githubusercontent.com/67608876/154546881-d63c645e-8833-46b2-9d52-2da36a0f6bf6.png) | ![Screenshot from 2022-02-17 23-19-17](https://user-images.githubusercontent.com/67608876/154546965-93df918a-e505-430e-915d-b0e891e3b0da.png) |

- [ ] Please kindly add a proper `heading` via using the `<hx/>` tag to this section. As a future web developer, you will have to build UI based on the design specified in the technical requirements, let's get used to it from the scratch :relaxed: :relaxed:

Reply:
- [ ] [OPTIONAL] I see that you have added the missing `hx`, however, you used a different font. Please ensure you stick to the design as much as possible.
- [ ] [OPTIONAL] Please note that the list of language need more padding to look like the one provided in the template and the fonts also differ. Kindly fix this.
![](https://...image1.png) | ![](https://...image2.png)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

![Screenshot from 2022-02-18 10-50-16](https://user-images.githubusercontent.com/58692590/154663691-cc18892e-ddb0-4181-9c95-8e26ede22024.png) | ![Screenshot from 2022-02-18 11-12-35](https://user-images.githubusercontent.com/58692590/154663729-9cc32240-00bd-46a1-94a0-27ed0a2075f8.png)